content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -150,6 +155,11 @@ metadata:
 spec:
   source:
     volumeSnapshotContentName: test-content
+status:
+  boundVolumeSnapshotContentName: test-content
+  creationTime: "2023-02-20T02:41:12Z"
+  readyToUse: true
+  restoreSize: 1Gi
```

Comment:
I wouldn't include a `status` here; it feels misleading to have it.

Reply:
@ameade Sorry for the long delay, but would you mind removing `status` from this manifest? `status` is not part of the desired state of the VolumeSnapshot.

Reply:
Agreed that we should not include "status" in a yaml file example.